### PR TITLE
Update user map for Aug 2017 #8056

### LINKED ISSUE #8056: Update user map for Aug 2017
Similar to #7424 

Institutions to add:

Australia | Monash University
-- | --
Australia | Queensland University of Technology
Australia | University of New South Wales
Australia | University of Melbourne
Brazil | IFRJ
Brazil | Universidade Catolica de Brasilia
Brazil | University of Brasilia
Canada | Cégep du Vieux-Montréal
Canada | Centennial College
Canada | Centennial College
Canada | Fleming College
Canada | Ryerson University
Colombia | Universidad del Norte
Finland | Pihlajamäen ala-asteen koulu
Germany | Hertie School of Governance
Germany | Philipps-Universität Marburg
Germany | Stuttgart Media University
Malaysia | HELP University
Malaysia | Universiti Tunku Abdul Rahman
Malaysia | University of Malaya
Mauritius | Afrian Leadership University
Mexico | UAS
Netherlands | Eindhoven University of Technology
Netherlands | Leiden University College
New Zealand | Eastern Institute of Technology
New Zealand | Massey University
New Zealand | University of Auckland
New Zealand | University of Auckland
New Zealand | University of Canterbury
Norway | Høgskolen i Oslo og Akershus
Philippines | De la Salle University
Philippines | Miriam College
Singapore | CWSS
Singapore | Nanyang Polytechnic
Singapore | Ngee Ann Polytechnic
Singapore | Singapore Institute of Technology
Singapore | Singapore Polytechnic
Singapore | SUTD
South Africa | University of Johannesburg
Sweden | Jönköping International Business School
Thailand | Thammasat University
Turkey | Credemy
UK | City of London School for Girls
UK | University College London
UK | University of Bradford
UK | University of Portsmouth
USA | AGS
USA | Arizona State University
USA | Bettendorf High School
USA | Bryant University
USA | C/3-13IN Basic Training
USA | CCSD59
USA | Central Oregon Community College
USA | Chaffey College
USA | CSU Sacramento
USA | Elon University
USA | Florida International University
USA | George Washington University
USA | Georgia Tech
USA | Georgia Tech
USA | Harvey Mudd College
USA | Indiana State University
USA | Indiana State University
USA | Jackson Heights Middle School
USA | Medical College of Wisconsin
USA | Medical College of Wisconsin
USA | Miami University
USA | Miami University
USA | Michigan State University
USA | Michigan State University
USA | Mississippi State University
USA | Montclair High School
USA | Nevada State College
USA | New England Insitute of Technology
USA | New Mexico State Univeristy
USA | North Carolina State University
USA | Northeastern University
USA | Northern Arizona University
USA | Northwestern College
USA | PACE
USA | Pitzer College
USA | Presbyterian Christian School
USA | San Diego State University
USA | Seattle Academy of Arts and Sciences
USA | Southeast Community College
USA | Southern Illinois University Edwardsville
USA | Spring Hill College
USA | University at Albany
USA | University at Albany, SUNY
USA | University if Minnesota
USA | University of Alabama at Birmingham
USA | University of California Berkeley
USA | University of Dallas
USA | University of Florida
USA | University of Kentucky
USA | University of Maryland, College Park
USA | University of Massachusetts Boston
USA | University of Minnesota
USA | University of North Carolina at Charlotte
USA | University of South Dakota
USA | University of Texas at Arlington
USA | University of Texas at Austin
USA | University of the Incarnate Word
USA | University of Washington
USA | University of West Florida
USA | West Chester University
USA | Western Carolina University
USA | Western Kentucky University
USA | William Penn University
USA | Worcester State University

### PR DESCRIPTION
**Pull request by g3chenyigmailcom**
added new institutions to userMapData.json

Fixes #8056

**Outline of Solution**
developed a powershell script to add new institutions to userMapData.json with sorted order and duplicates removed.

### CODE REVIEW

**Comment by whipermr5** [comment]
Hello @g3chenyigmailcom, welcome to TEAMMATES! Do read the checklist in step 3 of [process.md#step-3-fix-the-issue](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-3-fix-the-issue).

**Comment by g3chenyigmailcom** [comment]
Corrected json style :)

File: docs/setting-up.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ More information can be found at [this documentation](https://help.github.com/ar
 
 You are encouraged, but not required, to use an IDE to assist many development tasks.
 
-We currently support two IDEs: Eclipse IDE (full support) and IntelliJ IDEA (experimental; limited support).
+We currently support two IDEs: Eclipse IDE (full support) and IntelliJ IDEA.
```
Comment (whipermr5) [inline docs/setting-up.md]: Avoid fixing more than one issue in one PR.

File: src/main/webapp/data/userMapData.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "lastUpdated": "End of July 2017",
+  "lastUpdated": "End of September 2017",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: This is now the only change I see in this file?

**Comment by g3chenyigmailcom** [comment]
Hi John,

Greetings!
Sorry for the mistakes. I am new to GitHub and this is my first time
contributing to an open source project.
Should I delete my forked repo and re-start everything? And how about the
pull request, should I close it and start a new one as well?

Thanks,
Yi

**Comment by whipermr5** [comment]
No worries! There's no need to delete your forked repo; you can git reset your master branch to before your first commit and force push. I think this PR will automatically close when you do that. You can open another PR after starting from a new branch thereafter :)

File: docs/setting-up.md (hunk 1 above)
Comment (dipzzzz) [reply docs/setting-up.md]: what is the full code for this issue?

**Pull request closed without merging**